build.sh

```diff
@@ -222,7 +222,13 @@ function buildEventStore {
     patchVersionFiles
     patchVersionInfo
     rm -rf bin/
-    xbuild src/EventStore.sln /p:Platform="Any CPU" /p:DefineConstants="$DEFINES" /p:Configuration="$CONFIGURATION" || err
+    msbuild \
```

Comment:
you can use .net core sdk to build also `net461`.
so in the build script you can use `dotnet build`, `dotnet pack`, etc. who are easier to use (and same command line experience for users).

That also mean you dont need to use `msbuild` of mono for compiling, so you can use mono just as runtime.
It make it easier to upgrade (the .net sdk can be installed side by side, easier to upgrade and pin with a `global.json`)

Reply:
Hi @enricosada! Thanks for looking over this and providing feedback. We're actually using `dotnet msbuild` at the moment internally, but the build scripts are due for a rewrite once the rest of the cleanup work is done, so that hasn't been ported into the old ones yet (the change you are commenting on will not be in the eventual version of this pull request).

The only slightly weird thing appears to be that to build with Mono it is necessary to set `FrameworkPathOverrides` (or something similar, I forget the exact variable and am on a Windows box right now so don't have it in history).

Ultimately we want to fit into the "standard" .NET workflow as much as possible given the native dependencies.